apps/src/aichat/views/aichatView.module.scss

```diff
@@ -19,15 +19,18 @@
   .instructionsArea {
     width: 30%;
     height: 100%;
```

Comment:
I wonder if we should also prevent buttons from disappearing if user resizes window vertically.

Reply:
It's also included in this PR. If user resizes window vertically and height is too small - scroll for y axis will appear

Reply:
https://github.com/code-dot-org/code-dot-org/assets/22244040/76bb45e2-94bd-482e-b0c5-bfba56522765

Reply:
Nice! Thanks for the demo.